src/AppEndToEnd.test.tsx

```diff
@@ -221,21 +221,27 @@ it('basic end-to-end flow', async () => {
 
     // Vote for candidate contest
     if (title === presidentContest.title) {
-      fireEvent.click(getByText(presidentContest.candidates[0].name))
+      const candidateButton = getByText(presidentContest.candidates[0].name)
+      fireEvent.pointerDown(candidateButton)
+      fireEvent.pointerUp(candidateButton)
+      fireEvent.click(candidateButton)
     }
     // Vote for yesno contest
     else if (title === measure102Contest.title) {
-      fireEvent.click(getByText('Yes'))
+      fireEvent.pointerDown(getByText('Yes'))
     }
 
-    fireEvent.click(getByText('Next'))
+    const button = getByText('Next')
+    fireEvent.pointerDown(button)
+    fireEvent.pointerUp(button)
+    fireEvent.click(button)
```

Comment:
Are these necessary here, or are you just trying to make sure both work? Should that be its own test so as not to be confusing here?

Reply:
Perhaps an exhaustive standalone `Button` test which confirms only the expectations of the button actions should exist? 

All the other tests can then remain the same (using `onClick`).

Reply:
If we're going to explicitly support both for the whole flow, we probably want to make sure we test e2e flows with both approaches. I can imagine we change the component, for example, and then lose the behavior we need in a way that's invisible for tests.

Maybe have the same e2e test run twice, once with pointerDown, once with click.

Reply:
Running twice isn't necessarily a bad idea.